### PR TITLE
Renaming View model

### LINKED ISSUE #46: Rename View model
Rename `view` model to `view_by` and prevent confusion with django's views.py

### PR DESCRIPTION
**Pull request by Shalevro2**
Renaming View model

Renaming all the instances of View model to View_Article

Renaming test_view.py to test_view_article.py

Resolve #46

### CODE REVIEW

**Review by finziamit — COMMENTED** [review]
Maybe you should also rename `0007_test_view_data.py` to `0007_test_view_article_data.py`

**Review by rawadabu — APPROVED** [review]

File: projboard/tests/test_view_article.py [hunk 1]
```diff
@@ -36,15 +36,15 @@ def create_articles(self, create_user, create_subjects):
...
     def test_create_view(self, create_view):
-        assert create_view in View.objects.all()
+        assert create_view in View_Article.objects.all()
 
     @pytest.mark.django_db
     def test_amount_of_views_article(self, create_view):
-        amount_views = View.amount_of_views_article(create_view.article_id)
-        assert len(View.objects.filter(article_id=create_view.article_id)) == amount_views
+        amount_views = View_Article.amount_of_views_article(create_view.article_id)
+        assert len(View_Article.objects.filter(article_id=create_view.article_id)) == amount_views
```
Comment (Anas-mhameed) [inline projboard/tests/test_view_article.py:50]: could use Count function instead of len.

**Review by Anas-mhameed — APPROVED** [review]
LGTM

File: projboard/tests/test_view_article.py (hunk 1 above)
Comment (Shalevro2) [reply projboard/tests/test_view_article.py:50]: You right, but this PR just renaming the View model, I didn't changed any method.
Please resolve this conversation.

**Comment by Shalevro2** [comment]
> Maybe you should also rename `0007_test_view_data.py` to `0007_test_view_article_data.py`

Done at [3f7eaf0](https://github.com/redhat-beyond/Smarticle/pull/52/commits/3f7eaf05eba750e086cc5d2862d542df9c0e26df)

**Review by finziamit — APPROVED** [review]
LGTM

**Review by yaelnoyman1 — APPROVED** [review]
LGTM

File: projboard/tests/test_view_article.py (hunk 1 above)
Comment (Yarboa) [reply projboard/tests/test_view_article.py:50]: @Shalevro2 @Anas-mhameed 
You could agree of opening new issue mention it here
And continue with this Review

**Review by Yarboa — COMMENTED** [review]
LGTM

File: projboard/tests/test_view_article.py (hunk 1 above)
Comment (finziamit) [reply projboard/tests/test_view_article.py:50]: @Yarboa 
Opened issue #53 for it

**Comment by avi-biton** [comment]
LGTM

**Review by avi-biton — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

File: projboard/tests/test_view_article.py (hunk 1 above)
Comment (Yarboa) [reply projboard/tests/test_view_article.py:50]: @finziamit Please feel free to submit a fix also :) it is simple

**Pull request merged**